pkg/types/platform.go

```diff
@@ -53,6 +53,7 @@ type Platform struct {
 	Description       string       `json:"description"`
 	Credentials       *Credentials `json:"credentials,omitempty"`
 	OldCredentials    *Credentials `json:"old_credentials,omitempty"`
+	Version           string       `json:"-"`
```

Comment:
don't we want it returned in the platforms API? same question for "active" (i know it was there before)

Reply:
Are active and version field allowed to be changed via API ?